content/2023-07-31-t8code_version_141_released.md

```diff
@@ -0,0 +1,24 @@
+Title: t8code-1.4.1 released 
+Date: 2023-07-31 14:00 
+Category: News 
+Authors: David Knapp
+
+We have released version 1.4.1 of t8code!
+
+New features include:
+
+- A curved geometry for triangles
+
+- A generally improved geometry implementation
+
+- A parallel reader for vtk files
+
+- A function to create the connectivity for a cmesh
+
+- Improved testing
```

Comment:
If possible I would like to know what this improvement is, a short sentence might be nice. same for the next one, improved CI